django-drf-react-quickstart/mcbcapp/frontend/src/components/IntroPage.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+//import logo from "../assets/icons/logo.png";
+
+//console.log(logo);
+
+class IntroPage extends Component{ 
+    render(){
+               
+            const backgroundStyle = {
+                backgroundColor: "#32669d", 
+                width: "1919.9px", 
```

Comment:
i don't think you need to specify height and width - otherwise on screens bigger than these itll look like a blue box with a thick white border

Reply:
if it doesn't work without them just use 100% for both

Reply:
Having troubles in filling the height of the page with the background. Similar issue to: https://stackoverflow.com/questions/46769706/css-100-height-not-working-in-react-app/46770041
Could you have a look